components/suggestion/suggestion_box.jsx

```diff
@@ -237,7 +242,16 @@ export default class SuggestionBox extends React.Component {
         }, delay);
     }
 
+    preventSuggestionListClose() {
```

Comment:
Actually, I was going to ask the same thing. Did you guys talk elsewhere to resolve this?

Reply:
We talked about it through MM, forgot to update the thread with the conclusions. It's a small optimization to avoid the arrow function being created every time the render method is called. In this case it doesn't have a special impact on the performance, but was done as good practice.

Reply:
It still shouldn't get recreated as long as the arrow function is in the body of the class like
```javascript
class SuggestionBox {
    preventSuggestionListClose = () => {}

    render() {}
}
```
That should be roughly equivalent to binding the method in the constructor. Although, if that does get recreated every time render is called, then we've been telling people to make that change incorrectly, so it would be good to know if we're wrong with that

Reply:
Sorry, I misunderstood you guys, thought that as it was a simple function, you meant an **inline** arrow function. An arrow function in the body of the class should work perfectly fine. Fixing